coachmark/src/main/java/com/mercadolibre/android/andesui/coachmark/view/walkthroughscrolless/CoachmarkScrollessContainerView.kt

```diff
@@ -0,0 +1,49 @@
+package com.mercadolibre.android.andesui.coachmark.view.walkthroughscrolless
+
+import android.content.Context
+import androidx.constraintlayout.widget.ConstraintLayout
+import android.util.AttributeSet
+import android.view.View
+import com.mercadolibre.android.andesui.coachmark.R
+import com.mercadolibre.android.andesui.coachmark.view.CoachmarkOverlay
+import com.mercadolibre.android.andesui.typeface.getFontOrDefault
+import kotlinx.android.synthetic.main.andes_walkthrough_scrolless_container.view.*
+
+class CoachmarkScrollessContainerView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : ConstraintLayout(context, attrs, defStyleAttr) {
+
+    private var listener: CoachmarkContainerListener? = null
```

Comment:
```suggestion
    private var coachMarkContainerListener: CoachmarkContainerListener? = null
```
evita nombres genéricos